views/widget.scout.admin.php

```diff
@@ -20,30 +20,51 @@
 			$api_secret = $sailthru['sailthru_api_secret'];
 			$client = new WP_Sailthru_Client( $api_key, $api_secret );
 
-			// Get Sections
-			$sections = $client->apiGet('section');
 
-			// get sections
-			if (is_array ( $sections )) {
+			// Check the user has access to SPM. 
+			$settings = $client->apiGet('settings');
+			// set to be disabled by default for a safe fallback
+			$spm_enabled = false;
 
-				$section_dropdown = '<select name="'.$this->get_field_name( 'sailthru_spm_section' ).'">';
-				$section_dropdown .= '<option value="">-- Select --</option>';
-				
-				foreach ( $sections as $list ) {
-					foreach ($list as $section) {
-						if ($section['section_id'] == $section_id) {
-							$selected = ' selected';
-						} else {
-							$selected = '';
+			// Get the SPM settings
+			if (isset ($settings['features']['spm_enabled']) && $settings['features']['spm_enabled']) {
+				$spm_enabled = $settings['features']['spm_enabled'];
+			} 
+
+			// Get Sections
+			try {
+
+				if ($spm_enabled) {
```

Comment:
Line 30 will only execute is `spm_enabled` is 1, so pretty redundant. You should remove `if ($spm_enabled)` and just do 
```php 
if (isset ($settings['features']['spm_enabled']) && $settings['features']['spm_enabled']) 
```